linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -0,0 +1,117 @@
+use super::super::Concrete;
+use linkerd_app_core::{proxy::http, svc, Addr, Error, Result};
+use linkerd_distribute as distribute;
+use linkerd_http_route as http_route;
+use linkerd_proxy_client_policy as policy;
+use std::{fmt::Debug, hash::Hash, sync::Arc};
+
+pub(crate) mod backend;
+pub(crate) mod filters;
+
+pub(crate) use self::backend::Backend;
+pub use self::filters::errors;
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub(crate) struct Matched<M, P> {
+    pub(super) r#match: http_route::RouteMatch<M>,
+    pub(super) params: P,
+}
```

Comment:
it might be nice if there was a comment here explaining that this is used both for per-route layers and for per-backend layers within a route --- it would make reading the rest of the code much easier if we just stated that somewhere.